stackcollapse-phpspy.pl

```diff
@@ -56,21 +59,24 @@ sub usage {
 # internals
 my %stacks;
 my @frames;
+my $last_depth;
 
 while (defined(my $line = <>)) {
-    next unless $line =~ /^\d/;
+    next unless $line =~ /^\S \S/;
 
     my ($depth, $func) = (split ' ', $line)[0,1];
+    $last_depth = $depth;
 
-    if (@frames && $depth == 0) {
+    if (@frames && $depth eq '#') {
```

Comment:
if possible, it seems like it might be nice to treat `/^\s*#/` as comments and not treat them special on the collapsing side. that way we can add more meta information down the line.

Reply:
this also gets us around the issue above where a comment might not be printed. i'm working on a PR for this file -- maybe just easier to merge this now and i'll make that change then?